packages/components/src/components/list-box/_list-box.scss

```
@@ -751,7 +758,7 @@ $list-box-menu-width: rem(300px);
 
   .#{$prefix}--list-box__menu-item--active:hover,
   .#{$prefix}--list-box__menu-item--active.#{$prefix}--list-box__menu-item--highlighted {
-    background-color: $hover-selected-ui;
+    background-color: $selected-ui;
     border-bottom-color: $hover-selected-ui;
```

Should border-bottom-color change, too?
I don't know, but it just looks funny to have 2 different colors here?
```suggestion
    background-color: $selected-ui;
    border-bottom-color: $selected-ui;
```

---

Updated :) also I just your comment about `aria-current="true"` I will be updating that shortly

---

Thanks, @andreancardona. I'm not 100% sure how well `aria-current="true"` is supported in comboboxes, so after you put it in, and fix up `aria-selected="true"` to match the aria-activedescendant, please ping me and I will try it again in all the places to see if the `aria-current="true"` is helpful or just confusing.